testing/trino-tests/src/test/java/io/trino/tests/tpch/TestTpchConnectorTest.java

```diff
@@ -144,19 +144,19 @@ public void testPreSortedInput()
         // TPCH connector produces pre-sorted data for orders and lineitem tables
         assertExplain(
                 "EXPLAIN SELECT * FROM orders ORDER BY orderkey ASC NULLS FIRST LIMIT 10",
-                "\\QLimitPartial[10, input pre-sorted by (orderkey)]");
+                "\\QLimitPartial[count = 10, preSortedInputs = [orderkey]]");
```

Comment:
This is less user friendly. Why did it change?

Reply:
pretty print should most likely be part of text renderer, but then question is why we need NodeRepresentation at all (if text rendered needs to understand node type anyway)?

Maybe we keep identifier as is for now?

Reply:
I've updated the `preSortedInputs` field to `inputPreSortedBy`.

But if you are talking about the overall text plan. I don't find it less user-friendly compared to the current one because with this change, while we solved for JSON representation, I think we also enhanced the text plan.

1. We added more structure to the plan. Previously, only the details part of the plan was structured, while the identifier (or descriptor) part was plain text, thus different for every node.

2. More readability: I think having a user-friendly field name in the plan descriptor makes it easier for a new user to understand its meaning. For instance, in the case of `LocalExchange[SINGLE]` it was hard to figure out what `SINGLE` means. But now, with the specific field name `LocalExchange[partitioning = SINGLE]`, we can easily tell that it's related to partitioning.